src/components/CommunicationAndPartners/ContactForm/index.tsx

```diff
@@ -196,22 +196,20 @@ interface ContactFormProps {
     id: string | undefined;
     onHideAddContactModal: () => void;
     onAddContactCache: MutationUpdaterFn<CreateContactMutation>;
-    country?: string;
-    defaultCountry?: CountryOption[] | undefined | null;
+    defaultCountryOption?: CountryOption[] | undefined | null;
```

Comment:
```suggestion
    defaultCountryOption?: CountryOption | undefined | null;
```
Accept single value not list of options